iceoryx_hoofs/include/iceoryx_hoofs/cxx/string.hpp

```diff
@@ -673,6 +631,60 @@ IsStdStringOrCharArrayOrChar<T, bool> operator>=(const T& lhs, const string<Capa
 /// @return the stream output of the fixed string
 template <uint64_t Capacity>
 inline std::ostream& operator<<(std::ostream& stream, const string<Capacity>& str) noexcept;
+
```

Comment:
Can you please move the above `operator<<`  above the `AXIVION DISABLE STYLE AutosarC++19_03-A13.5.5:` line? The reason is that operator does not violate the AUTOSAR rule and all the comparison operators are grouped together.

Reply:
Can you please explain why you reject this before resolving the conversation?

Reply:
@FerdinandSpitzschnueffler may I have missed the comments, not intended. I'll address this on next update.